src/index.tsx

```diff
@@ -34,6 +35,37 @@ export const GET_ALL_POSTS = gql`
   }
 `;
 
+export const GET_POST_DETAILS = gql`
```

Comment:
Does the gql call need to live in index.tsx?

Reply:
It is best practice for it to live there, yeah